### ISSUE TITLE
Info, Privacy & Co

### ISSUE THREAD

**Issue by sheepyhollow** [zentrumnawi/solid-frontend#974]
Sammel-Issue für Korrekturen an den Info-Seiten
* Planty
  - [ ] HS-Geisenheim in Info-Seite erwähnen
 * AIS
   - [ ] "Diversität der Organismen" aus Infotext entfernen
 * SOLID allgemein
   - [ ] Info & Datenschutz vereinen (Logo könnte ein rundes Split-Icon mit Info und Paragraphenzeichen sein)

**Comment by sheepyhollow** [comment]
@kerstinhartmann Hier sollten ggf. noch weitere Aufgaben hinzugefügt werden.

**Comment by sheepyhollow** [comment]
Ist es möglich, beim Routing direkt einen Tab anzusprechen?

**Comment by kerstinhartmann** [comment]
GeoMat/Planty/AIS
- [ ] "Über die App"/Textanpassung 
--> Mit dem neuen Mediamodel können wir nicht nur Bildmaterial (wie momentan beschrieben) sondern auch Video- und Audiomaterial einbinden. Das könnte hier ergänzt werden. 
Vorschlag: "s.o.l.i.d. ist eine Plattform für die strukturierte Darstellung von Objekten. Sie bringt Bild-, Video- und Audiomaterial sowie objekttypische Merkmale in übersichtlicher Form zusammen."
- [ ] "Urheberrecht"/Typo "den Medien" statt "~~dem~~"
- [ ] "Urheberrecht" 
"Bitte nutzen Sie das Feedback-Formular für Rückfragen. " ggf. entfernen, da direkt im Anschluss "Kontakt & Feedback" folgt - erneut mit dem Hinweis auf das **Kontaktformular**. Wenn es aus rechtlichen Gründen bleiben soll, dann einheiltliche Bezeichnung "**Kontaktformular**" wählen (und klickbar gestalten).

AIS
- [ ] "Kontakt & Feedback" 
--> "Für Rückmeldungen und -fragen nutzen Sie..." statt <del>"Rückmeldungen und Rückfragen"</del>
- [ ] Kontaktformular optisch hervorheben
--> Das Wort "Kontaktformular" ist klickbar, aber das ist für den User optisch momentan nicht zu erkennen.

GeoMat 
- [ ] "Über die App"/Textanpassung 
"**GeoMat**_digital_ ist der Ursprund des E-Learning-Projekts s.o.l.i.d ..." statt "~~GeoMat ist..~~"
- [ ] Ziele der App/Textanpassung
"Die Lernanwendung **GeoMat**_digital_ ist <del>als Lernanwendung </del> an die Lehrveranstaltung _Geomaterialien_ am ..."
- [ ] Einheitliche Schreibweise **GeoMat**_digital_  (fett, kursiv) 
- [ ] Überschrift "Haftungsausschluss" Doppelpunkt entfernen
- [ ] "Kontakt & Feedback" 
--> "Für Rückmeldungen und -fragen nutzen Sie..." statt <del>"Rückmeldungen und Rückfragen"</del>

Planty 
- [ ] "Ansprechpartner"/ Wort "Kontaktformular" optisch hervorgeben s.o.

**Comment by anhnguyen-98** [comment]
@sheepyhollow Es geht hier drum, dass die `Datenschutz` kann als ein Tab in `Info` wie News und Changelog wäre oder?

**Comment by sheepyhollow** [comment]
Genau! Einen Icon-Entwurf für die Landing Page haben ich schon irgendwo.

Es war noch etwas unsicher, ob das rechtlich in Ordnung ist - es gibt eine Regel, die sagt, dass Info und Datenschutz mit maximal zwei Klicks erreichbar sein müssen. Aber ich denke, wenn die Tabs gut sichtbar benannt sind, geht das.

**Comment by sheepyhollow** [comment]
Wir hätten dann drei Tabs: Info | Privacy | News  (+ Changelog -> siehe #1238)

**Comment by sheepyhollow** [comment]
Soll ich dann noch mit den Anpassungen im Text (Kommentar von Kerstin) noch warten bis es einen PR gibt?

**Comment by anhnguyen-98** [comment]
Ja wartest mal bitte noch bis ich einen PR hochlade. Damit wir wenig Konflikt haben.

**Comment by sheepyhollow** [comment]
Gut! Dann mache ich das als Contribution zu Deinem PR (und nicht in einem eigenen).

**Comment by sheepyhollow** [comment]
Der Link zur Datenschutzerklärung ganz am Ende der Info Component funktioniert jetzt nicht mehr weil privacy nur noch ein Tab ist und keine eigene URL mehr hat.

Ist es möglich, eine Funktion zu bauen, mit der der Tab gewechselt wird? Das sollte noch in #1367 gemacht werden.

Wenn es nicht möglich ist, nehmen wir den Hinweis auf die Datenschutzerklärung raus.

**Comment by anhnguyen-98** [comment]
> Der Link zur Datenschutzerklärung ganz am Ende der Info Component funktioniert jetzt nicht mehr weil privacy nur noch ein Tab ist und keine eigene URL mehr hat.
> 
> Ist es möglich, eine Funktion/Link zu bauen, mit der der Tab gewechselt wird? Das sollte noch in #1367 gemacht werden.
> 
> Wenn es nicht möglich ist, nehmen wir den Hinweis auf die Datenschutzerklärung raus.

Wie ich recherchiert habe, gibt es noch keinen Weg, die Events von `ngComponentOutlet` also die dynamische Component, hier ist die `info-page-content.component` von jeweiligen Apps zu der `info.component` in Skeleton zu senden. Wir brauchen dieses Event, um die Tab in `mat-tab-group` zu wechseln.
Da habe ich 2 Vorschlagen:

1.  Wir nehmem den Hinweis auf die Datenschutzerklärung in `info-page-content.component` raus.
2. Wir verschieben den den Hinweis auf die Datenschutzerklärung in `info-page-content.component` auf die `info.component` in Skeleton (siehe das Bild). Damit wir da direkt das Event bekommen und die Tab wechseln können.

<img width="791" alt="Screenshot 2022-01-16 at 11 10 26" src="https://user-images.githubusercontent.com/71195698/149655908-43b83ba8-ec59-47cd-969b-22923e3c1240.png">

Was hältest du davon?

**Comment by sheepyhollow** [comment]
Hm... ich will eigentlich nicht so viel Aufwand machen 😂 Aber ich fände 2. doch charmant.
Gerade weil wir mit der kombinierten Info/Privacy-Component so ein bisschen am Rande der gesetzlichen Anforderung sind. 

Ich habe es bei mir ausprobiert (funktioniert natürlich nicht, weil noch eine Funktion hinein muss, die den Tab-Index anspricht) 
und ich finde den Vorschlag gut - es braucht nur noch ein bisschen Styling (margins).
